Websites/bugs.webkit.org/js/field.js

```diff
@@ -25,6 +25,9 @@ function validateEnterBug(theform) {
     var description = theform.comment;
     var attach_data = theform.data;
     var attach_desc = theform.description;
+    var submitbug = document.getElementById('submitbug');
```

Comment:
This would cause the JS for the default template to throw since that ID doesn't exist I believe. Not sure we care about keeping support for the default  template

Reply:
IMO no, because the custom template overrides it. The custom template would have to be deleted for the default template to take effect.